bitnami/rabbitmq-cluster-operator/values.yaml

```diff
@@ -575,6 +581,12 @@ msgTopologyOperator:
     ##
     pullSecrets: []
 
+  ## @param msgTopologyOperator.watchAllNamespaces Watch for resources in all namespaces
+  ##
+  watchAllNamespaces: true
+  ## @param msgTopologyOperator.watchNamespaces [array] Watch for resources in the given namespaces
+  ##
+  watchNamespaces: []
```

Comment:
Same comment as for the cluster operator section

> I would add a comment stating that watchNamespaces is ignored if watchAllNamespaces is true. This may sound obvious, but one could interpret "watch all namespaces in this list" 🙂

Reply:
Good catch!